demos/sjtwo/ltc4150/source/main.cpp

```diff
@@ -0,0 +1,53 @@
+#include "L1_Peripheral/lpc40xx/gpio.hpp"
+#include "L2_HAL/sensors/battery/ltc4150.hpp"
+#include "utility/log.hpp"
+#include "utility/time.hpp"
+#include "utility/units.hpp"
+
+bool ApproxEquality(float expected, float actual, float resolution)
+{
+  return -resolution < (actual - expected) && (actual - expected) < resolution;
+}
+
+int main()
+{
+  constexpr float kResolution = 0.001f;
+  LOG_INFO("Ltc4150 application starting...");
+  // Creating GPIO on pin 2.0
+  sjsu::lpc40xx::Gpio tick_pin(2, 0);
+  // Creating GPIO on pin 2.1
+  sjsu::lpc40xx::Gpio polarity_pin(2, 1);
+  // Set the sense resistor resistance to be 50 milliohms. This will provide a
+  // 50 millivolt drop at maximum charge or discharge to invoke an interrupt on
+  // the tick_pin.
```

Comment:
Most of this does not make sense with regards to how the chip works. Remove everything except for this line "// Set the sense resistor resistance to be 50 milliohms."

Reply:
Done.